### DISCUSSION TITLE
Add additional catalog views under a library..?

### DISCUSSION THREAD

**Discussion by spinjector** [dbeaver/dbeaver#14456] [General]
In `Database Navigator`, can additional database object types be added under a connection/library..?

For instance, most servers have `Tables`, `Views`, `Indexes`, and `Procedures`. But I'm using IBM DB2 , which has dozens of additional catalog views, and I would like to see `Functions` without having to run a query. I looked through the [wiki](https://github.com/dbeaver/dbeaver/wiki/Database-Navigator) and googled around, but I haven't found any substantial information to indicate if it's even possible or not..

Like so:
![dbeaver_2019-09-11_13-27-28](https://user-images.githubusercontent.com/21039987/64724065-b4387c00-d49f-11e9-9209-1bdac21641f1.png)

**Comment by JMarkMurphy** [comment]
Functions are in the Procedures list. When you select the routine, it shows the routine type (Function or Procedure), and the result type, so you can even see if it is a table function. In addition, procedures and functions have a different icon, so you can see by looking at the list which are procedures, and which are functions. 

It would be nice to have yet another icon for table functions to differentiate those.
